manifests/init.pp

```diff
@@ -462,7 +466,7 @@
   Optional[String] $service_overrides_template        = $docker::params::service_overrides_template,
   Optional[Boolean] $service_hasstatus                = $docker::params::service_hasstatus,
   Optional[Boolean] $service_hasrestart               = $docker::params::service_hasrestart,
-
+  Optional[String] $registry_mirror                   = $docker::params::service::registry_mirror,
```

Comment:
@MWilsonPuppet can you review this line, it's failing the unit tests

Reply:
@davejrt Hey Dave I dug in and  issue was with  login in ERB template. I have made changes to the erb template and tested working also added a negative test case. Ill push  changes now. Just need to add a final change to read me .